src/Inciter/Transporter.C

```diff
@@ -329,7 +332,7 @@ Transporter::load( uint64_t nelem )
 }
 
 void
-Transporter::partition()
+Transporter::part()
```

Comment:
What benefit is there in renaming partition to part? Based upon the description neither name was particularly meaningful but ``partition'' contained much more semantic meaning